packages/dso-toolkit/components/Componenten/tooltip/tooltip.njk

```diff
@@ -1,13 +1,13 @@
 <!-- Impliciet: Geen @for attribute dus target is direct parent -->
 <button type="button">
   <dso-icon icon="plus"></dso-icon>
-  <dso-tooltip position="{{ position }}">
+  <dso-tooltip position="{{ position }}" small="{{ small }}">
```

Comment:
`small` behandelen zoals alle HTML5 boolean attributen werken is fraaier. Denk aan `disabled`.

Reply:
En laten we daar nou die handige `attributes()` functie voor hebben